src/hotspot/share/opto/gcm.cpp

```diff
@@ -924,36 +1051,54 @@ Block* PhaseCFG::insert_anti_dependences(Block* LCA, Node* load, bool verify) {
       // This turns off the process of gathering non_early_stores.
     }
   }
-  // (Worklist is now empty; all nearby stores have been visited.)
+  // Worklist is now empty; we have visited all possible anti-dependences.
 
   // Finished if 'load' must be scheduled in its 'early' block.
   // If we found any stores there, they have already been given
   // precedence edges.
```

Comment:
Might be clearer since we always talked about anti-dependency edges while the concept to implement them are precedence edges.
```suggestion
  // anti-dependency edges.
```

Reply:
Yes, best to be consistent. Fixed (although I changed to anti-dependence, see other comment above).